examples/with-remix-thirdpartyemailpassword/app/lib/superTokensTypes.ts

```diff
@@ -0,0 +1,48 @@
+import { SessionContainerInterface } from "supertokens-node/lib/build/recipe/session/types";
+
+export type PartialRemixRequest = {
+  method: string;
+  url: string;
+  headers: Headers;
+  formData: () => Promise<FormData>;
+  json: () => Promise<unknown>;
+  cookies: {
+    getAll: () => { name: string; value: string }[];
+  };
+};
+
+export type HTTPMethod = "post" | "get" | "delete" | "put" | "options" | "trace";
+
+export interface ExtendedSession extends SessionContainerInterface {
```

Comment:
why is this type needed? The `session` object being returned from getSession has the type of `SessionContainerInterface` which is good enough

Reply:
Removed.